Mathlib/Algebra/Module/Injective.lean

```diff
@@ -435,3 +436,36 @@ lemma Module.Injective.extension_property
   (Module.Baer.of_injective inj).extension_property f hf g
 
 end lifting_property
+
+section injective_products
+
+theorem pi (R : Type) [Ring R] {ι : Type} (M : ι → Type)
```

Comment:
```suggestion
theorem pi (R : Type*) [Ring R] {ι : Type*} (M : ι → Type*)
```
`R : Type*` means that `R` is a type in an arbitrary universe, while `Type` is syntax sugar for `Type 0`, the "zero"-th universe.